### DISCUSSION TITLE
Migrating from Xamarin Forms 5.0 to MAUI, Which Version of Prism?

### DISCUSSION THREAD

**Discussion by wadairjlmarinesystems** [PrismLibrary/Prism#2989] [.NET MAUI]
We are migrating from Xamarin Forms 5.0 to MAUI. 

Should we migrate to .NET 7 and Prism 8.1.273-pre and wait for .NET 8 to mature? Or should we go strait to Prism 9.0.271-pre and .NET 8?

**Comment by dansiegel** [comment]
While we had an early preview for net6/7... only net 8 for .NET MAUI is considered Production ready.
